framework/framework.go

```diff
@@ -131,7 +131,7 @@ func (c *Contract) SendTransaction(method string, args []interface{}, confidenti
 		panic(err)
 	}
 	if receipt.Status == 0 {
-		panic("bad")
+		panic(fmt.Errorf("status not correct"))
```

Comment:
ideally, errors would not be dynamic, but exported

Reply:
curious that the linter doesn't complain

Reply:
This is not an error per se but just some formatting.

This will be addressed alongside https://github.com/flashbots/suapp-examples/pull/14#discussion_r1425418951